product_docs/docs/transporter/ba_preview/getting_started/installing/index.mdx

```diff
@@ -6,32 +6,28 @@ navigation:
   - linux_x86_64
 ---
 
-To install EDB Transporter for internal purposes:
+Select a link to access the applicable installation instructions.
 
-### Accessing CloudSmith
+## Linux [x86-64 (amd64)](linux_x86_64)
 
-Log in to Okta, and navigate to the CloudSmith tile to access the dev CloudSmith repository. 
+### Red Hat Enterprise Linux (RHEL) and derivatives
 
-### Finding the appropriate package
+- [RHEL 8](linux_x86_64/transporter_rhel_8), [RHEL 7](linux_x86_64/transporter_rhel_7)
```

Comment:
We have dropped support for RHEL 7 for all of our products, so I would recommend adding docs for RHEL 8 and 9, if possible. Otherwise, let's include RHEL 8 only.

Reply:
Removed RHEL 7 and added support for RHEL 8.